apiserver/facades/client/cloud/cloud.go

```diff
@@ -105,9 +105,9 @@ func NewCloudAPI(
 	}, nil
 }
 
-func (api *CloudAPI) canAccessCloud(ctx context.Context, cloud string, user string, access permission.Access) (bool, error) {
+func (api *CloudAPI) canAccessCloud(ctx context.Context, cloud string, user names.UserTag, access permission.Access) (bool, error) {
 	id := permission.ID{ObjectType: permission.Cloud, Key: cloud}
```

Comment:
Choose a better name for this now that we're passing `user.Id(), id` to the method.

Reply:
Changed to userID and cloudID to make it clearer. The type for will be changed back to username type in https://github.com/juju/juju/pull/17775